cardscan-base/src/main/java/com/getbouncer/cardscan/base/SSDOcrDetect.java

```diff
@@ -96,6 +100,63 @@ private String runModel(Bitmap image) {
         return number;
     }
 
+    public interface OnDetectListener {
+        void complete(String result);
+    }
+
+    private static <T> T getKeyByMaxValue(Map<T, ? extends Comparable> map) {
+        Map.Entry<T, ? extends Comparable> maxEntry = null;
+        for (Map.Entry<T, ? extends Comparable> entry : map.entrySet()) {
+            if (maxEntry == null || entry.getValue().compareTo(maxEntry.getValue()) > 0) {
+                maxEntry = entry;
+            }
+        }
+
+        if (maxEntry != null) {
+            return maxEntry.getKey();
+        } else {
+            return null;
+        }
+    }
+
+    public static void runInCompletionLoop(
```

Comment:
where does `runInCompletionLoop` get called?

Reply:
this gets called from the `LivenessCheckActivity` in the cardverify repo.